### PR TITLE
Addon resizer release 1.8

### PR DESCRIPTION
**Pull request by kawych**
Fast-forward release-1.8 branch to release-1 branch (release 1.8.1 was created from release-1 branch).

### CODE REVIEW

File: addon-resizer/nanny/apis/nannyconfig/v1alpha1/defaults.go [hunk 1]
```diff
@@ -39,3 +39,18 @@ func SetDefaults_NannyConfiguration(obj *NannyConfiguration) {
 		obj.MemoryPerNode = "0"
```
Comment (x13n) [inline addon-resizer/nanny/apis/nannyconfig/v1alpha1/defaults.go]: They doesn't seem to be reasonable defaults.

**Comment by x13n** [comment]
One general comment: Why do you need master, release-1, release-1.7 and release-1.8 branches at the same time? Do you really support 4 different versions simultaneously?

File: addon-resizer/nanny/apis/nannyconfig/v1alpha1/defaults.go (hunk 1 above)
Comment (kawych) [reply addon-resizer/nanny/apis/nannyconfig/v1alpha1/defaults.go]: I think it is reasonable, it makes it clear if one specifies only cpu per node and doesn't pass memoryPerNode. Any other default would be really confusing IMO.

File: addon-resizer/README.md [hunk 2]
```diff
@@ -91,6 +91,10 @@ spec:
           command:
             - /pod_nanny
             - --config-dir=/etc/config
+            - --cpu=300m
```
Comment (x13n) [inline addon-resizer/README.md]: So... heapster will be using 300m + 20m per node now? Also, why is nanny using 300m/200Mi?

File: addon-resizer/nanny/apis/nannyconfig/v1alpha1/defaults.go (hunk 1 above)
Comment (x13n) [reply addon-resizer/nanny/apis/nannyconfig/v1alpha1/defaults.go]: Sorry, I didn't see that actual "defaults" are elsewhere. Ignore this comment :-)

**Pull request merged**